.pre-commit-config.yaml

```diff
@@ -5,3 +5,16 @@
     - id: git-check  # Configure in .gitattributes
     - id: git-dirty  # Configure in .gitignore
     - id: shellcheck
+- repo: https://github.com/pre-commit/mirrors-mypy
+  sha: v0.701
+  hooks:
+    - id: mypy
+- repo: https://github.com/pre-commit/mirrors-isort
+  sha: v4.3.20
+  hooks:
+    - id: isort
+- repo: https://github.com/pre-commit/pre-commit-hooks/
+  sha: v2.2.3
+  hooks:
+    - id: flake8
```

Comment:
you'll want to use flake8 from the official repo, using flake8 from pre-commit-hooks is deprecated,

```yaml
-   repo: https://gitlab.com/pycqa/flake8
    rev: 3.7.7
    hooks:
    -   id: flake8
```